run_common.go

```diff
@@ -208,6 +208,39 @@ func (b *Builder) generateHostname(rdir, hostname string, chownOpts *idtools.IDP
 	return cfile, nil
 }
 
+// createSSLCertFile creates a containers CA cert file
+func (b *Builder) createSSLCertFile(rdir, containerPath string, chownOpts *idtools.IDPair) (string, error) {
+	resolvedSSLCertPath, err := util.ResolveRootCACertFile()
+	if err != nil {
+		return "", fmt.Errorf("error resolving cert file on host: %w", err)
+	}
+
+	hostCertBytes, err := os.ReadFile(resolvedSSLCertPath)
+	if err != nil {
+		return "", fmt.Errorf("error reading cert file on host: %w", err)
+	}
+
+	cfile := filepath.Join(rdir, filepath.Base(containerPath))
+	if err = ioutils.AtomicWriteFile(cfile, hostCertBytes, 0o644); err != nil {
+		return "", fmt.Errorf("writing %s into the container: %w", containerPath, err)
+	}
```

Comment:
I don't know how big the certs files can be but generally it seems best to copy it without reading the full thing into memory.